src/bfx_tools/samtools.ml

```diff
@@ -150,6 +154,7 @@ let sort_bam_no_check ~(run_with:Machine.t) ?(processors=1) ~by input_bam =
     | `Read_name -> "sort" :: "-n" :: command
   in
   do_on_bam ~run_with input_bam ~product ~make_command
+    ~more_requirements:[`Memory `Big]
```

Comment:
Yes it is. It is the temporary way of saying "a lot of mem". One day we will try to find out the exact requirements, but that requires a lot of work. 
(memory usage is not even always a monotonous function of input size, for example, maximal coverage in a region of a bam can make tools create big pileups, or big gaps between paired reads, etc.)